resources/application-connector-nginx/templates/upgrade-job.yaml

```diff
@@ -0,0 +1,35 @@
+#Job that allows upgrade of Kyma from version 0.7 to 0.8
+#In 0.8 nginx LoadBalancer is extracted to _this_ chart, but still binds to the same IP Address. This creates a conflict.
+#This job deletes existing nginx LoadBalancer (installed by 0.7) so that the new one can be provisioned.
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ .Release.Name }}-loadbalancer-migration
+  namespace: kyma-system
+spec:
+  template:
+    metadata:
+      name: {{ .Release.Name }}-loadbalancer-migration
+      namespace: kyma-system
+      annotations:
+        sidecar.istio.io/inject: “false”
+    spec:
+      serviceAccountName: {{ .Release.Name }}-loadbalancer-migration-service-account
+      restartPolicy: OnFailure
+      containers:
+      - name: {{ .Release.Name }}-loadbalancer-migration
+        image: eu.gcr.io/kyma-project/develop/xip-patch:1103850e
+        command:
+          - bash
+          - -c
+          - |
+            set -e
+            SERVICE_TO_REMOVE="application-connector-nginx-ingress-controller"
```

Comment:
Shouldn't it be the name of the service? If so it is not accurate as the service name changed to `application-connector-nginx-nginx-ingress-controller`

Reply:
This is OK. We have to remove old service here - and it is the old name. This job is only for migration from "old" (0.7) to "new" (0.8) and shouldn't be used for anything else, hence the name is hardcoded.